kitsune/wiki/templates/wiki/mobile/document.html

```diff
@@ -44,6 +44,9 @@
         {{ _('Share this article: {link}')|f(link=link|safe) }}
       </p>
     {% endif %}
+    <p>
+      Last Update: {{ latest_approved_revision.created.date() }}
```

Comment:
you can just change this to:
```
document.current_revision.created.date()
```

and then undo all the changes to `wiki/views.py`